Configuration/PyReleaseValidation/README.md

```diff
@@ -61,6 +61,8 @@ The offsets currently in use are:
 * 0.7: trackingMkFit modifier
 * 0.701: DisplacedRegionalStep tracking iteration for Run-3
 * 0.702: trackingMkFit modifier for Phase-2 (initialStep only)
+* 0.703: LST tracking, initialStep+HighPtTripletStep only, on CPU
+* 0.704: LST tracking, initialStep+HighPtTripletStep only, on GPU
```

Comment:
Some questions and comments about these workflows, as implemented in [`Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py`](https://github.com/cms-sw/cmssw/pull/45117/files#diff-8c163203cbaa64f0e0e6aa2b16346cfab3eaf84a57a008c7fa37193e8b9b2adb).

For the Run 3 alpaka-based workflows, we do not have a distinction of "cpu" vs "gpu" workflows: the same workflows (_e.g._ `xxx.402`) should work on CPU-only, on NVIDIA GPUs and on AMD GPUs.

@cms-sw/reconstruction-l2 @cms-sw/pdmv-l2 @cms-sw/upgrade-l2  do you think it is useful to have different workflows for different backends ?
If yes, we should do the same for the Run 3 workflows.
If not, we shouldn't do it for the LST workflows, either.

Reply:
Currently:
  - `.703` force running on CPU with `--accelerators cpu`
  - `.704` force running on NVIDIA GPUs with `--accelerators gpu-nvidia`
  
(see below)

By the way, the CUDA-based workflows like `.5xx` are going to be removed soon (either by the last 14.1.x prerelease or early in the following release cycle), so they shouldn't be used as a recerence.